crates/forge/src/lib.rs

```diff
@@ -295,6 +348,68 @@ fn run_tests_from_file(
     })
 }
 
+fn run_with_fuzzing(
+    runner_config: &RunnerConfig,
+    contracts: &HashMap<String, StarknetContractArtifacts>,
+    predeployed_contracts: &Utf8PathBuf,
+    runner: &SierraCasmRunner,
+    case: &TestCase,
+    args: &Vec<&ConcreteTypeId>,
+    fuzzer: &mut fuzzer::Random,
+) -> Result<(TestCaseSummary, u32)> {
+    if contains_non_felt252_args(args, &BUILTINS) {
```

Comment:
This will never happen since you filter out the non-felt252 arguments earlier

Reply:
Good point